src/main/java/bio/terra/stairway/Database.java

```diff
@@ -172,8 +150,8 @@ public void complete(FlightContext flightContext) {
             connection.setAutoCommit(false);
 
             statement.setString("output_parameters", flightContext.getWorkingMap().toJson());
-            statement.setBoolean("succeeded", flightContext.getResult().isSuccess());
-            statement.setString("error_message", flightContext.getResult().getErrorMessage());
+            statement.setString("status", flightContext.getFlightStatus().name());
```

Comment:
Stairway doesn't use Spring at all. Since I want to leave the door open for multiple Stairways, I didn't want to use any of the singleton annotations. I would have to make the stairway database an @Repository to get the spring @Transaction stuff to work.

I could have used Spring's JdbcTemplate, but it seemed safer to not rely on Spring at all. Instead, I implemented a version of named parameters so that the SQL style and parameter access would be uniform.

We can certainly revisit this sometime down the road.